Extension/src/stealth/instrument.ts

```diff
@@ -327,7 +327,16 @@ function logCall(instrumentedFunctionName, args, callContext, _logSettings) {
 /**
  * Provides the properties per prototype object
  */
-Object.getPrototypeByDepth = function (subject, depth) {
+
+declare global {
+  interface Object {
+    getPrototypeByDepth(subject: string | undefined, depth: number): any
+    getPropertyNamesPerDepth(subject: any, maxDepth: number): any
+    findPropertyInChain(subject, propertyName)
+  }
+}
+
+Object.prototype.getPrototypeByDepth = function (subject, depth) {
```

Comment:
Finally found a way how to declare the extension methods on Object.
Going to do this for everything to please the typechecker